cluster-service/cspr/namespace-cleaner.yaml

```diff
@@ -0,0 +1,77 @@
+---
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: namespace-cleaner
```

Comment:
Should it be named orphaned-namespace-cleaner?

Reply:
Yeah, okay, I'll change it to that.